collect_app/src/test/java/org/odk/collect/android/backgroundwork/SyncFormsTaskSpecTest.kt

```diff
@@ -50,19 +50,30 @@ class SyncFormsTaskSpecTest {
     }
 
     @Test
-    fun `getTask calls synchronize with projectId`() {
+    fun `when LAST_UNIQUE_ATTEMPT equals true getTask calls synchronize with proper data`() {
```

Comment:
Instead of `proper data` in these tests it could be `with/without notify` or `with notify true/false`

Reply:
Fixed.